quickwit/quickwit-search/src/search_job_placer.rs

```diff
@@ -177,13 +178,30 @@ impl SearchJobPlacer {
         let mut job_assignments: HashMap<SocketAddr, (SearchServiceClient, Vec<J>)> =
             HashMap::with_capacity(num_nodes);
 
+        let total_load: usize = jobs.iter().map(|job| job.cost()).sum();
+
+        // allow arround 1% disparity. Round up so we never end up in a case where
+        // target_load * num_nodes < total_load
+        // some of our tests needs 2 splits to be put on 2 different searchers. It makes sens for
+        // these tests to keep doing so (testing root merge). Either we can make the allowed
+        // difference stricter, find the right split names ("split6" instead of "split2" works).
+        // or modify mock_split_meta() so that not all splits have the same job cost
+        // for now i went with the mock_split_meta() changes.
+        const ALLOWED_DIFFERENCE: usize = 101;
```

Comment:
1% is a. bit tight? Is this coming from your simulations?

```suggestion
        const ALLOWED_DIFFERENCE: usize = 101;
```

Reply:
it seems to be good when splits are roughly equal, but a bit low when there are wide variation. I'm going to increase it a bit